sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientTestBase.java

```diff
@@ -7,6 +7,7 @@
 import com.azure.ai.textanalytics.models.DetectLanguageInput;
```

Comment:
Consider setting StepVerifier.defaultTimeout so that it is not infinite. And resetting it when the test class finishes.

Reply:
Updated.